leapp/utils/audit/__init__.py

```diff
@@ -283,7 +283,7 @@ def get_audit_entry(event, context):
 
     :param context: The execution context
     :type context: str
-    :param event: Type of this event e.g. process-start or process-end but can be anything
+    :param event: Type of this event e.g. process-start or process-result but can be anything
```

Comment:
Now that I see this, this is weird wording. Type of this event ... can be anything. It sounds like it can be of any type, which it can't. Identifier for this type of event, might be more clear.

Reply:
Makes sense. Fixed.